pkg/app/shim/shim.go

```diff
@@ -66,22 +69,29 @@ func NewCommand() *cobra.Command {
 
 // runE implements the actual command logic
 func runE(cmd *cobra.Command, args []string) error {
+	cmd.Printf("Running %s version: %s\n", BinaryName, GitTag)
 	// there should be at least one argument (the deployer) unless the user
 	// is asking for help on the shim itself
 	if len(args) < 1 {
 		return cmd.Help()
 	}
 
-	// gracefully handle -h or --help if it is the only argument
+	// gracefully handle help or version command if it is the only argument
```

Comment:
Unrelated to this feature, but if other arguments are set AND --help is included, it will try to deploy?

Reply:
no https://github.com/kubernetes-sigs/kubetest2/blob/96ab8c9c5ca183b362a8f95d24bd328e2527ab70/pkg/app/cmd.go#L139-L143